CHANGELOG.md

```diff
@@ -1,4 +1,28 @@
 ## 2.16.0 (Unreleased)
+### Notes:
+- Supported Controller version: **UserConnect-6.1** (tested on **UserConnect-6.1.1078**)
+- Supported Terraform version: **v0.12.x**
+
+### Features:
+1. Implemented new resource to support periodic ping:
+  - **aviatrix_periodic_ping**
+2. Implemented new resource to support fqdn pass through:
+  - **aviatrix_fqdn_pass_through**
+3. Implemented support for specifying and updating ``gateway1_excluded_network_cidrs``, ``gateway1_excluded_tgw_connections``, ``gateway2_excluded_network_cidrs``, and ``gateway2_excluded_tgw_connections`` for **aviatrix_transit_gateway_peering**
+4. Implemented support for configuring ``bgp_polling_time``, ``prepend_as_path``, ``local_as_number``, and ``bgp_ecmp`` for **aviatrix_transit_gateway**
+5. Implemented support for ``enable_vpc_dns_server`` in **aviatrix_controller_config**
+6. Implemented support for changing name servers individually on **aviatrix_geo_vpn**
```

Comment:
6. Implemented support for updating name servers individually on ELBs under the **aviatrix_geo_vpn**